packages/nextjs/src/server/errors.ts

```diff
@@ -29,10 +29,20 @@ Alternatively, you can set your own ignoredRoutes. See https://clerk.com/docs/ne
 `;
 
 export const getAuthAuthHeaderMissing = () =>
-  'You need to use "authMiddleware" (or the deprecated "withClerkMiddleware") in your Next.js middleware file. You also need to make sure that your middleware matcher is configured correctly and matches this route or page. See https://clerk.com/docs/quickstarts/get-started-with-nextjs';
+  `Clerk: getAuth() was called but it looks like you aren't using authMiddleware in your middleware file. Please ensure the following:
+- Use "authMiddleware" (or the deprecated "withClerkMiddleware") in your middleware file.
+- Make sure your middleware matcher is configured correctly and it matches this route or page.
+- If you are using the /src directory, ensure that the middleware.ts file is located under it.
+For more details, see https://clerk.com/docs/quickstarts/get-started-with-nextjs.
+`;
 
 export const authAuthHeaderMissing = () =>
-  "Clerk: auth() was called but it looks like you aren't using `authMiddleware` in your middleware file. Please use `authMiddleware` and make sure your middleware matcher is configured correctly and it matches this route or page. See https://clerk.com/docs/quickstarts/get-started-with-nextjs";
+  `Clerk: auth() was called but it looks like you aren't using authMiddleware in your middleware file. Please ensure the following:
```

Comment:
Please make the `auth()` `getAuth()` part dynamic and consolidate these two messages to save the extra bytes

Reply:
Done